demo/sdl_opengl2/nuklear_sdl_gl2.h

```diff
@@ -74,6 +75,10 @@ nk_sdl_render(enum nk_anti_aliasing AA)
     int display_width, display_height;
     struct nk_vec2 scale;
 
+    float now = ((float)SDL_GetTicks64()) / 1000;
+    sdl.ctx.delta_time_seconds = now - sdl.time_of_last_frame;
```

Comment:
When the renderer is initialized, should we prime `time_of_last_frame`? Or does that matter?